plasmapy/physics/dielectric.py

```diff
@@ -0,0 +1,199 @@
+"""Functions to calculate plasma dielectric paramaters"""
+
+from astropy import units
+import plasmapy.utils as utils
+
+from ..constants import (pi, m_e, c, mu0, e, eps0)
+from .parameters import gyrofrequency, plasma_frequency
+
+r"""
+Values should be returned as an Astropy Quantity in SI units.
+"""
+
+
+@utils.check_quantity({
+    'B': {'units': units.T, 'can_be_negative': False},
+    'omega': {'units': units.rad/units.s, 'can_be_negative': False},
+})
+def cold_plasma_permittivity_SDP(B, species, n, omega):
+    """
+    Magnetized Cold Plasma Dielectric Permittivity Tensor Elements.
+    Elements (S, D, P) are given in the "Stix" frame, ie. with B // z.
+
+    The :math:`\exp(-i \omega t)` time-harmonic convention is assumed.
+
+    References
+    - T.H. Stix, Waves in Plasma, 1992.
+
+    Parameters
+    ----------
+    B : ~astropy.units.Quantity
+        Magnetic field magnitude in units convertible to tesla.
+
+    species : List of string
+        List of the plasma particle species
+        e.g.: ['e', 'D+'] or ['e', 'D+', 'He+'].
+
+    n : List of ~astropy.units.Quantity
+        List of species density in units convertible to per cubic meter
+        The order of the species densities should follow species.
+
+    omega : ~astropy.units.Quantity
+        Electromagnetic wave frequency in rad/s
+
+    Returns
+    -------
+    S : ~astropy.units.Quantity
+        S ("Sum") dielectric tensor element
+
+    D : ~astropy.units.Quantity
+        D ("Difference") dielectric tensor element
+
+    P : ~astropy.units.Quantity
+        P ("Plasma") dielectric tensor element
```

Comment:
Some more description here would be nice :)
  
EDIT: I guess this is fine with the examples below. No more phone reviewing!